Sources/SecureXPC/Server/XPCAnonymousServer.swift

```diff
@@ -33,6 +33,13 @@ internal class XPCAnonymousServer: XPCServer {
             endpoint: xpc_endpoint_create(self.anonymousListenerConnection)
         )
     }
+    
+    // This exists for testing purposes
+    internal func shutdown() {
```

Comment:
Perhaps:

```suggestion
    internal func simulateDisconnectionForTesting() {
```

Reply:
Seems reasonable. Have made this change.

Overall I dislike having test only code in the product itself, but there's a massive complexity jump to actually spinning up a multiprocess integration test to not need this.

Reply:
It might be possible to move this into the test module, using `@testable import` to gain access to otherwise inaccessible `internal` APIs.

But if it's using any `private` APIs, then it won't be possible (I'm on mobile right now so it's hard for me to check this)

Reply:
Ah yeah it'd need access to `anonymousListenerConnection` which is private. I'll leave it as-is for now, but happy to revisit this.